packages/sdk/src/swap/types.ts

```diff
@@ -62,13 +62,15 @@ export type SwapStatusResponse = CommonStatusFields &
         swapId: string;
         depositAmount: string;
         depositReceivedAt: number;
+        depositAddressCreatedAt: number;
```

Comment:
think you can move it to `CommonStatusFields` 🙂

Reply:
and it needs to be `number | undefined` i guess

Reply:
its not great that we dont check if the response matches the types we have anywhere 🙈